ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/ConfigurationHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using ServiceWebsite.AcceptanceTests.Configuration;
+using ServiceWebsite.AcceptanceTests.Contexts;
+using ServiceWebsite.Configuration;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class ConfigurationHelper
+    {
+        private const string INDIVIDUAL_TEST_ACCOUNT = "video-hearing-booking-creation-individual@hmctest.com";
```

Comment:
Should be PascalCase for private const members

Reply:
done